HISTORY.md

```diff
@@ -6,6 +6,21 @@ dev
 
 -   \[Short description of non-trivial change.\]
 
+**Dependencies**
+
+- Instead of `chardet`, use the MIT-licensed `charset_normalizer` for Python3
+  to remove license ambiguity for projects bundling requests. If `chardet`
+  is already installed on your machine it will be used instead of `charset_normalizer`
+  to keep backwards compatibility.
+
+  You can also install `chardet` while installing requests by
+  specifying `[use_chardet_on_py3]` extra as follows:
+
+  ```shell
+  pip install "requests[use_chardet_on_py3]"
```

Comment:
Something went wrong with that suggestion .. the ending triple quotes disappeared somehow, try this

```suggestion
    ```shell
    pip install "requests[use_chardet_on_py3]"
    ```
```